packages/start/entry-server/entries.ts

```diff
@@ -0,0 +1,49 @@
+import { renderToStringAsync, renderToStream } from "solid-js/web";
+import { RequestContext } from "./StartServer";
+
+export function renderAsync(fn: (context: RequestContext) => any, options?: any) {
+  return () => async (context: RequestContext) => {
+    let markup = await renderToStringAsync(() => fn(context), options);
+    if (context.routerContext.url) {
+      return Response.redirect(new URL(context.routerContext.url, context.request.url), 302);
+    }
+
+    context.responseHeaders.set("content-type", "text/html");
+
+    return new Response(markup, {
+      status: 200,
+      headers: context.responseHeaders
+    });
+  };
+}
+
+function handleRedirect(context) {
```

Comment:
Is this supposed to be unimplemented?

Reply:
It was causing infinite loops. So I left it for now.